### DISCUSSION TITLE
[BUG] Mux player crushed in Nextjs 14.0.2

### DISCUSSION THREAD

**Discussion by zrtsky** [muxinc/elements#817] [Q&A]
#### Screen
<img width="1035" alt="Screenshot 2023-11-10 at 10 53 32" src="https://github.com/muxinc/elements/assets/42916643/45ec05e3-3325-4c67-bd0c-9981cc14dd77">

#### Logs

<img width="638" alt="Screenshot 2023-11-10 at 10 56 38" src="https://github.com/muxinc/elements/assets/42916643/27827070-601a-487d-829e-6b5163ceeef9">

### PS

I think it's can be started from `next: 14.0.2-canary.0`, here is the [link](https://github.com/vercel/next.js/issues/57964#issuecomment-1791801804) to discussion about this issue

**Comment by luwes** [comment]
https://github.com/muxinc/elements/issues/818#issuecomment-1846334884
